src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -312,9 +313,9 @@ public ItemStack removeItemNeeded(@NotNull final ItemStack stack)
             if (stack.isItemEqualIgnoreDurability(neededItem))
             {
                 //todo make this sofisticated as soon as material handling has been implemented.
-                //final int itemsToRemove = Math.min(neededItem.getCount(), stackCopy.getCount());
-                //neededItem.setCount(stackCopy.getCount() - itemsToRemove);
-                //stackCopy.setCount(stackCopy.getCount() - itemsToRemove);
+                //final int itemsToRemove = Math.min(ItemStackUtils.getSize(neededItem), ItemStackUtils.getSize(stackCopy));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)